EXAMPLES.rst

```diff
@@ -1,10 +1,10 @@
 Projects using Sphinx
 =====================
 
-This is an (incomplete) alphabetic list of projects that use Sphinx or
-are experimenting with using it for their documentation.  If you like to
-be included, please mail to `the Google group
-<https://groups.google.com/forum/#!forum/sphinx-users>`_.
+This is an incomplete list of projects that use Sphinx for their documentation.
+If you like to be included, please create a `pull request on GitHub
```

Comment:
```suggestion
If you would like to be included, please create a `pull request on GitHub
```

(or perhaps, "if you would like a project to be included", or something along those lines)